hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -116,3 +122,13 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
       
       
 
+
+class JSBuilderSanityCheck extends JSBuilder:
+  override def handleFunction(ps: List[semantics.Param], b: Block)(using Raise, Scope): (Document, Document) =
+    val params = ps.map(p => Scope.scope.allocateName(p.sym))
+    val paramsStr = doc"args_" :: params.mkDocument("_")
```

Comment:
What if this name conflicts with an already-defined name in the source? The proper thing to do here is to use a `TempSymbol` that you make sure to allocate in the scope and that you then `lookup_!`. There does not need to be one such symbol per function. You can create a single one at the top-level of `JSBuilderSanityChecks`.